File: buildSrc/src/main/groovy/org/mockito/release/exec/Exec.java
package org.mockito.release.exec;

import java.io.File;

/**
 * Process execution services
 */
public class Exec {

    /**
     * Provides process runner for given working dir
     */
    public static ProcessRunner getProcessRunner(File workDir) {
        return new DefaultProcessRunner(workDir);
    }
}

File: buildSrc/src/main/kotlin/org/mockito/release/exec/exec.kt
package org.mockito.release.exec

import java.io.File

/**
 * Process execution services
 */
object Exec {

    /**
     * Provides process runner for given working dir
     */
    fun getProcessRunner(workDir: File): ProcessRunner {
        return DefaultProcessRunner(workDir)
    }
}